### PR TITLE
Updated Read me for eq-google-slides-to-schema 

### PR DESCRIPTION
**Pull request by yoganandkunche**
Updated Read me for eq-google-slides-to-schema

python convert.py --presentation_id=1dBACWN8uOdznEpJHYL9zQQAiFoHyVe-eLVdJwc2ZGvw --out=mwss_4.json

### CODE REVIEW

**Comment by collisdigital** [comment]
Please squash and sort out the line endings changes.

File: README.md [hunk 1]
```diff
@@ -27,16 +27,16 @@ Specifically the section "Step 1: Turn on the Google Slides API"
...
 ```
 ./convert.py --help
 ```
 
-Example with the id of a Google Slides presentation to convert and 
+Example with the id of a Google Slides presentation to convert and
 output file:
 ```
-./convert.py --presentation_id=[...] --out=survey.json
+python /convert.py --presentation_id=[...] --out=survey.json
```
Comment (benjefferies) [inline README.md]: Shouldn't this be ./convert.py

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ to process. This is separate from the API access and is requested
 at runtime.
 
 ## Presentation Format
-In order to extract content from the Slides into a schema some 
+In order to extract content from the Slides into a schema some
```
Comment (benjefferies) [inline README.md:48]: Is it not better to one line this and let whatever is rendering it wrap?

**Review by benjefferies — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (yoganandkunche) [reply README.md:48]: out of scope for this PR

**Review by benjefferies — APPROVED** [review]

**Pull request merged**